src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -88,6 +88,15 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('password_restrictions')
+                ->addDefaultsIfNotSet()
+                    ->children()
+                        ->integerNode('min_digits')->defaultNull()->end()
+                        ->integerNode('min_uppercase')->defaultNull()->end()
+                        ->integerNode('min_special_characters')->defaultNull()->end()
+                        ->integerNode('min_length')->defaultNull()->end()
+                        ->integerNode('max_length')->defaultNull()->end()
```

Comment:
Why would you encourage anyone to set a maximum password length (by providing this option)? Please don’t! @jockri made a great point!